api/src/services/ssi-service.ts

```diff
@@ -209,7 +213,7 @@ export class SsiService {
 
 	generateIdentity() {
 		try {
-			const { doc, key } = new Document(this.config.keyType) as IdentityDocument;
+			const { doc, key } = new Document(this.config.keyType, Identity.Network.mainnet().toString()) as IdentityDocument;
```

Comment:
do they now hardcode the Node URL inside the lib?

Reply:
Oh yes they have pointing to the iota nodes, but I've wanted to replace them by the defined env var. I will adjust it.